### DISCUSSION TITLE
can you add mount network drive http://mediafire.com/

### DISCUSSION THREAD

**Discussion by CodeLyokoXtEAM** [Studio-42/elFinder#3305] [Ideas]
can you add mount network drive http://mediafire.com/  mange files in elfinder?

**Comment by nao-pon** [comment]
There is no plan at the moment. We would welcome the PR.

**Comment by CodeLyokoXtEAM** [comment]
@nao-pon I found useful site.

https://github.com/MediaFire/
https://github.com/MediaFire/mediafire-php-open-sdk
